tests/unit/grains/test_core.py

```diff
@@ -243,10 +243,11 @@ def _import_mock(name, *args):
                                 # Mock linux_distribution to give us the OS
                                 # name that we want.
                                 distro_mock = MagicMock(
-                                    return_value=('SUSE test', 'version', 'arch')
+                                    return_value=os_release_map['linux_distribution']
                                 )
                                 with patch("salt.utils.fopen", mock_open()) as suse_release_file:
-                                    suse_release_file.return_value.__iter__.return_value = os_release_map.get('suse_release_file', '').splitlines()
+                                    suse_release_file.return_value.__iter__.return_value = \
+                                        os_release_map.get('suse_release_file', '').splitlines()
```

Comment:
Doesn't looks like you've removed whole SUSE-related things from this function

Reply:
`_run_suse_os_grains_tests` also uses this `_run_os_grains_tests` function. Besides that this code part was also part of `_run_ubuntu_os_grains_tests`.